config.json.tmpl

```diff
@@ -0,0 +1,12 @@
+{
+    "refresh_period"        : $REFRESH_PERIOD,
+    "sleep_timeout"         : $SLEEP_TIMEOUT,
+    "notify_period"         : $NOTIFY_PERIOD,
```

Comment:
This line should be removed if we're going to rely on cron.

Or it should look like this (if we rely on internal implementation):
```json
    "notify_schedule"       : ["09:00", "21:00"],
```